### PR TITLE
[Gitlab] Fix pagination

### PR DESCRIPTION
**Pull request by paulnicolet**
## Description

Fixes an issue in the GitLab extension that was preventing fetching all the pages from the API, even though the `all` parameter was enabled.

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/gitlab/src/gitlabapi.ts
Comment (tonka3000) [inline extensions/gitlab/src/gitlabapi.ts:347]: Why limit to 50? The [GitLab docs](https://docs.gitlab.com/ee/api/index.html#offset-based-pagination) mention that 100 is max, so with 50 this will double the requests in some cases.

Comment (paulnicolet) [reply extensions/gitlab/src/gitlabapi.ts:347]: No particular reason. I was thinking that 100 page size was a "hack" to fake pagination. 

But indeed we can keep 50 when fetching a single page, and 100 when fetching them all to limit requests.

Comment (tonka3000) [reply extensions/gitlab/src/gitlabapi.ts:347]: Please a it back to reduce requests for larger data volumes.

**Review by metakirby5 — APPROVED** [review]
LGTM! (Not sure if I have approval power though?)

**Comment by pernielsentikaer** [comment]
Can you give it another spin @tonka3000

**Review by tonka3000 — APPROVED** [review]
I have no Mac right now to test it live, but code looks good to me.

**Review by pernielsentikaer — APPROVED** [review]
Then let's get it merged 🔥

**Pull request merged**
